docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
+  app:
+    image: rko619619/task_manager:latest
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWD: example
+      POSTGRES_HOST: db
+    ports:
+      - 3000:3000
+  db:
+    image: postgres:alpine3.15
+    environment:
+      POSTGRES_PASSWORD: example
```

Comment:
please, hide it somehow (.env or Rails.credentials)

Reply:
Fixed.